packages/ui-causal-graph-editor/tests/causal-graph-store.spec.tsx

```diff
@@ -146,5 +147,37 @@ describe('CausalGraphStore', () => {
             expect(state.graph?.nodes()).toEqual(['node1', 'node3']);
         });
     });
-
+    describe('Update extra metadata', () => {
```

Comment:
The changes were mostly in the parser/serializer layer, so I would expect this would be covered in the parsing/serializing test to make sure `input == serializer(parser(input))`

Reply:
There is already a test that covers that. To properly test serializing `extra` metadata we actually need to modify the graph. Moved it to the serializer test suite.

Reply:
Alright sure